app/routes/issuer.py

```diff
@@ -272,6 +274,151 @@ async def revoke_credential(
     bound_logger.info("Successfully revoked credential.")
 
 
+@router.post("/publish-revocations", status_code=204)
+async def publish_revocations(
+    publish_request: PublishRevocations,
+    auth: AcaPyAuth = Depends(acapy_auth),
+):
```

Comment:
I think it's good practice to add the return type, even if it's None (as signified by status_code=204)

So:
```py
@router.post("/publish-revocations", status_code=204)
async def publish_revocations(
    publish_request: PublishRevocations,
    auth: AcaPyAuth = Depends(acapy_auth),
) -> None
```

Can you please search for `status_code=204` in the codebase and update each endpoint with that code to specify return type `-> None`. I think there's still a few that don't specify it, but It's good for every endpoint to specify type